filesystem/src/main/java/org/frankframework/filesystem/FileSystemSender.java

```diff
@@ -103,7 +107,10 @@ public SenderResult sendMessage(Message message, PipeLineSession session) throws
 			Message result = actor.doAction(message, pvl, session);
 			return new SenderResult(result);
 		} catch (FileSystemException e) {
-			throw new SenderException(e);
+			String forwardName = e.getForward().getForwardName();
+
+			log.error("Error from FileSystemActor, will return forward name [{}]",forwardName, e);
```

Comment:
```suggestion
			log.info("error from FileSystemActor, will call forward name [{}]",forwardName, e);
```
Error level is wel heel erg bot.. dat hoeft toch niet?

Reply:
Je hebt helemaal gelijk. Ik weet niet waarom daar `log.error()` stond.